### DISCUSSION TITLE
Disable breakpoint(s) in debugger addon

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7237] [Ideas]
Add an enable/disable input to the breakpoint block of the debugger addon, so that you can temporarely enable or disable the breakpoint block without having to add/remove the block from the script you are debugging. 

Another useful addition could be adding a this/all input, so to disable all breakpoints if they are spread in several scripts of several sprites.

The final block would be "enable/disable this/all breakpoint"

**Comment by WorldLanguages** [comment]
The web Chrome Devtools have this feature:

![image](https://github.com/ScratchAddons/ScratchAddons/assets/17484114/daf68b4f-63b0-46c6-9aae-51c1b8ca2a3e)

**Reply by sfederici** [reply]
Yes, right. I would like to get the same possibility in the Scratch debugger addon

**Reply by DNin01** [reply]
We could also go even further and allow each breakpoint to be individually deactivated if need be, which is also a feature from Chrome's DevTools. Breakpoints could even be automatically disabled when other users view your project. The list that you would use to do this could also be used to jump to any of the breakpoints you have placed in your project.
